packages/wrangler/src/r2/object.ts

```diff
@@ -569,9 +569,20 @@ export const r2BulkPutCommand = createCommand({
 				}
 			);
 		} else {
+			// Cloudflare API rate limits
+			// 1200 requests per 5 minutes
+			// We add some headroom (100 requests) for other API usage
+			// ref: https://developers.cloudflare.com/fundamentals/api/reference/limits/
+			const API_RATE_LIMIT_WINDOWS_SEC = 5 * 60; // 5 minutes
```

Comment:
NIT: just have ms here and avoid having to compute it later:


```suggestion
			const API_RATE_LIMIT_WINDOWS_MS = 5 * 60 * 1000; // 5 minutes
```